tools/unicode_case_conversion.py

```diff
@@ -46,7 +217,11 @@ def parse_unicode_sequence(raw_data):
             continue
 
         # Convert it to unicode code point (from hex value without 0x prefix)
-        result += unichr(int(unicode_char, 16))
+        try:
+            result += unichr(int(unicode_char, 16))
+        except NameError:
+            result += chr(int(unicode_char, 16))
```

Comment:
I'm not sure about this, but: maybe we should extract the `int` call before the try-catch. What do you think?